docs/general/server/media/shows.md

```diff
@@ -58,7 +60,7 @@ Season folders shouldn't contain the series name, otherwise Jellyfin can in cert
 
 ## Show Specials
 
-Show specials can be added in the `Season 00` folder. If supported by your metadata provider those files will be matched. In case your metadata provider does not provide information about the special, it is recommended to use a name which describes the content of the special instead of naming it `Episode S00Exy.mkv`. This is done to avoid wrong metadata being pulled for the special and to provide a proper presentation.
+Show specials can be added in the `Season 00` folder. If supported by your metadata provider those files will be matched. In case your metadata provider does not provide information about the special, it is recommended to use a name which describes the content of the special instead of naming it `Series Name S00Exy.mkv`. This is done to avoid wrong metadata being pulled for the special and to provide a proper presentation.
```

Comment:
What is this change aiming to accomplish?

Reply:
Clarifying that an episode should always be prefixed with the series name. There have been instances where users tried to actually use the "Episode" moniker instead of the series name. Which does not work with the parser. Series names should be, if at all suffixed to an episode.